manager_helpers/dask_manager_template.py

```diff
@@ -0,0 +1,106 @@
+"""
+Dask Distributed Manager Helper
+
+Conditions:
+- Dask Distributed and Dask Job Queue (dask_jobqueue in Conda/pip)
+- Manager running on the head node
+- SLURM manager
+
+For additional information about the Dask Job Queue, please visit this site:
+https://jobqueue.dask.org/en/latest/
+"""
+
+# Fractal Settings
+# Location of the Fractal Server you are connecting to
+FRACTAL_ADDRESS = "localhost:7777"
+
+# How many cores per node you want your jobs to have access to
+CORES_PER_NODE = 0
+# How much memory per node (in GB) you want your jobs to have access to
+MEMORY_PER_NODE = 0
+# How many tasks per node you want to execute on
+MAX_TASKS_PER_NODE = 0
+# Maximum number of nodes to try and consume
```

Comment:
Try to reword slightly, `consume` doesn't really resonate too well I think.  use/utilize maybe?